packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1289,21 +1322,23 @@ class Auth {
      * A wrapper method for the SLAS endpoint: getPasswordResetToken.
      *
      */
-    async getPasswordResetToken(parameters: ShopperLoginTypes.PasswordActionRequest) {
+    async getPasswordResetToken(parameters: ShopperLoginTypes.getPasswordResetTokenBodyType) {
```

Comment:
Nit: I was thrown off a bit by `ShopperLoginTypes` having a lower case ie. `getPasswordResetTokenBodyType` while others like `GetPasswordLessAccessTokenParams` (the ones we've defined ourselves) are upper case.

The types in `ShopperLoginTypes` probably should have been upper case.

Reply:
Types should use PascalCase following TypeScript conventions, but we have the ShopperLoginTypes from commerce-sdk-isomorphic using camelCase like with `getPasswordResetTokenBodyType`. 

I agree, ideally we fix that in commerce-sdk-isomorphic to be consistent in future versions.